src/ragas/metrics/__init__.py

```diff
@@ -1,4 +1,5 @@
 from ragas.metrics.answer_relevance import AnswerRelevancy, answer_relevancy
+from ragas.metrics.answer_similarity import AnswerSimilarity, answer_similarity
```

Comment:
why are you exposing answer similarity? it should be `AnswerCorrectness` right?

Reply:
Answer similarity is an independent metric.  It can be used by the user if required.